arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Ior.kt

```diff
@@ -1326,21 +1187,25 @@ public fun <A, C, B : C> Ior<A, B>.widen(): Ior<A, C> =
 
 @Deprecated(
   NicheAPI + "Prefer using widen",
-  ReplaceWith("widen", "arrow.core.widen")
+  ReplaceWith("widen()", "arrow.core.widen")
 )
 public fun <AA, A : AA, B> Ior<A, B>.leftWiden(): Ior<AA, B> =
   this
 
 @Deprecated(
   NicheAPI + "Prefer using the inline ior DSL",
-  ReplaceWith("ior(SA::combine) { Pair(this@zip.bind(), fb.bind()) }", "arrow.core.raise.ior", "arrow.typeclasses.combine")
+  ReplaceWith(
+    "ior(SA::combine) { Pair(this.bind(), fb.bind()) }",
+    "arrow.core.raise.ior",
+    "arrow.typeclasses.combine"
+  )
 )
 public fun <A, B, C> Ior<A, B>.zip(SA: Semigroup<A>, fb: Ior<A, C>): Ior<A, Pair<B, C>> =
   ior(SA::combine) { Pair(this@zip.bind(), fb.bind()) }
 
 @Deprecated(
   NicheAPI + "Prefer using the inline ior DSL",
-  ReplaceWith("ior(SA::combine) { map(this@zip.bind(), c.bind()) }", "arrow.core.raise.ior", "arrow.typeclasses.combine")
+  ReplaceWith("ior({a, b -> a + b}) { map(this.bind(), c.bind()) }", "arrow.core.raise.ior")
```

Comment:
I think @serras has asked this in another PR, but being `A` a generic type, this code won't compile unless we run it under the scope of a `Semigroup` instance, will it? 

Additionally, why do you use the `SA::combine` function in this `ReplaceWith` [expression](https://github.com/arrow-kt/arrow/pull/2976/files#diff-c5990e586847dc3242060efe9f70a256299b84edf33bdb0171486a3db9c5184bR1198) and use the `plus` operator in the rest of `zip` alternatives?

Reply:
This was actually a bug 😅 . Good catch, @franciscodr !. See [`133847c` (#2976)](https://github.com/arrow-kt/arrow/pull/2976/commits/133847c46a24d3c2b6482009d962b0d4917fb62b)